workflows/toast_so_sim.py

```diff
@@ -623,6 +648,11 @@ def dump_spt3g(job, args, data):
             (ops.elevation_model.out_model, ops.elevation_model.out_model),
         ]
     )
+
+    print([
```

Comment:
Was this a stale debug print statement?

Reply:
I think is some leftover from the first time that I looked at how to export data and then completely forgot about it since I saved always in HDF5. Removed